Gems/LmbrCentral/Code/Source/Shape/EditorBaseShapeComponent.cpp

```diff
@@ -200,11 +200,12 @@ namespace LmbrCentral
 
     void EditorBaseShapeComponent::OnShapeColorChanged()
     {
-        m_shapeColor.SetA(AzFramework::ViewportColors::DeselectedColor.GetA());
+        AZ::Color drawColor(m_shapeColor);
+        drawColor.SetA(AzFramework::ViewportColors::DeselectedColor.GetA());
```

Comment:
question: Is this change definitely needed? I think I understand the other ones but do things work as expected with this change reverted? I don't quite follow how this change is different, though maybe it is something subtle I'm missing

Reply:
@Jackie9527 can you speak to this feedback from @hultonha ?

Reply:
The color of label would be modified, because its alpha value is changed to `0.4`, which is weird.
![image](https://user-images.githubusercontent.com/80555200/219943864-0dae2b02-ef02-40ce-bf0b-d5aca8438cff.png)
![image](https://user-images.githubusercontent.com/80555200/219943868-67f8c2f9-0807-4832-afde-0c099183a523.png)

Reply:
@hultonha Can you take a look at Jackie's response and see if it answers your question? Thanks!

Reply:
Sorry I missed this before, thanks for pinging me @cgalvan. Sorry @Jackie9527 I think I understand now, apologies I misunderstood the original code and see why the temporary is needed now to not modify the member variable. Looks good to me thanks for getting back to me 👍